closure/goog/testing/performancetimer.js

```diff
@@ -194,6 +194,7 @@ goog.testing.PerformanceTimer.prototype.runTask = function(task) {
  * Finishes the run of a task by creating a result object from samples, in the
  * format described in {@code run}.
  * @see goog.testing.PerformanceTimer#run
+ * @param {Array<number>} samples The samples to analyze.
```

Comment:
@shicks That's part of analyzerChecks which is off for Closure Library now: https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/DiagnosticGroups.java#L507.